apps/site/lib/site_web/router.ex

```diff
@@ -211,7 +211,7 @@ defmodule SiteWeb.Router do
 
     get("/alerts", AlertController, :show_by_routes)
     get("/stops/:stop_id/alerts", AlertController, :show_by_stop)
-    get("/stop/:stop_id/facilities", StopController, :get_facilities)
```

Comment:
question: I didn't notice any code removing `StopController.get_facilities/2`, did you mean to include that in this PR?